src/controllers/chatbotController.js

```diff
@@ -53,7 +53,7 @@ let postWebhook = (req, res) => {
 
             // Check if the event is a message or postback and
             // pass the event to the appropriate handler function
-            if (webhook_event.message) {
+            if (webhook_event.message && sender_psid != 100730141849563) {
```

Comment:
Thêm phần này để ko xử lý tin nhắn của bot gửi đến users nhé, trc đấy tin nhắn của bot gửi cũng bị handleMessage nên việc sử dụng biến toàn cục bị ảnh hưởng.

Reply:
t nghĩ là nên lưu nó thành 1 biến chứ ko nên để luôn là số thế này

Reply:
hmm ok